pages/documentation/guidance/accessibility.md

```diff
@@ -6,83 +6,167 @@ redirect_from:
+
+{:.border-top-1px.padding-top-2.measure-2}
+### Blind users
+- Support screen readers and other assistive technology like braille displays
+- Outline semantic landmarks and regions
+- Assure all content has clear context and labeling
+- Provide text equivalents for visual resources
+- Announce updates of page state changes
+- Trap and move focus logically and predictably
+- Avoid visual-exclusive meaningful functionality
+
+{:.border-top-1px.padding-top-2.margin-top-2.measure-2}
+### Limited-vision users
+- Design for readability
+- Design generous, forgiving interaction targets
+- Allow layouts to respond to user scaling settings and screen magnifiers
+- Prefer linear layouts
+- Don't rely on hover for critical information
+- Use legible visual contrast
+- Avoid color-exclusive meaningful functionality
+
+{:.border-top-1px.padding-top-2.margin-top-2.measure-2}
+### Deaf users
+- Provide accurate text equivalent for audio resources
+- Avoid audio-exclusive meaningful functionality
+
+{:.border-top-1px.padding-top-2.margin-top-2.measure-2}
+### Limited-mobility users
+- Support keyboard-only (linear) functionality and conventions
+- Support touchscreen (direct input) functionality and conventions
+- Design generous readability and interaction targets
+- Prefer linear layouts
+- Don't rely on hover for critical information
+
+{:.border-top-1px.padding-top-2.margin-top-2.measure-2}
+### Sensory-sensitive users
+- Keep it simple
+- Avoid flashing and repetitive animation
+- Avoid very high contrast
+- Avoid alarming colors and images
+- Avoid audio and video that start automatically
+
+## What the design system does
+USWDS provides accessible components, the guidance to implement these components, and the tools to customize and extend the design system accessibly.
+
+{:.border-top-1px.padding-top-2.measure-2}
+### Deliver accessible components
+- Test with screen readers (VoiceOver, NVDA)
+- Test with keyboard-only
+- Test with touch-only
+- Test across browsers and operating systems
+- Collect community feedback
+- Scan with automated tools, like pa11y and aXe, on every code change. If it doesn't pass, we can’t merge the code
+- Manually test components and functionality with a Trusted Tester
+- **If it isn't accessible, it won't ship.**
+
+{:.border-top-1px.padding-top-2.margin-top-2.measure-2}
+### Provide guidance on how to implement our components accessibly
+- Include an accessibility section with each component
+
+{:.border-top-1px.padding-top-2.margin-top-2.measure-2}
+### Include accessibility-focused tools to build new functionality
+- Colors with magic number contrast
+- Rem-based size tokens and an em-based media query mixin that respond to user scale settings
+- Normalized typography tokens that provide reliable, readable results regardless of the typeface
+
+{:.border-top-1px.padding-top-2.margin-top-2.measure-2}
+### Feature smart settings that allow accessible customization with minimal custom code
```

Comment:
It seems like you intend 'smart' here to mean something like 'context-aware', which might be a more specific way of describing the functionality.

Reply:
@sknep good catch -- let's change this to read: "Feature [DELETE smart and add] context-aware settings that allow accessible customization with minimal custom code"